code/datums/manufacturing.dm

```diff
@@ -1935,6 +1935,7 @@ proc/get_nice_mat_name_for_manufacturers(mat)
 /datum/manufacture/hat_ltophat
 	name = "Large Top Hat"
 	item_paths = list("FAB-1")
+	item_names = list("Fabric")
```

Comment:
Your branch is fucked up

Reply:
Shit, Will het to fixing it tomorrow

Reply:
Try `git rebase -i master` and then delete all the lines with the commits that shouldn't be in the branch.